### PR TITLE
Mobile version

### PR DESCRIPTION
**Pull request by lorenzoworx**
- Add markup for works and about sections
- Add styles for works and about sections

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -32,5 +32,127 @@ <h1 class="playfair">Okolo <br> Oshoke</h1>
...
+    <section class="mobile mobile-about">
+        <div>
+            <h1 class="playfair">About</h1>
+            <p class="poppins">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore 
+                magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod
+            </p>
+            <button type="button">
+                Get my CV
+                <span><img src="img/ic_download.svg" alt=""></span>
+            </button>
```
Comment (vechicin) [inline index.html:122]: This button doesn't have the `active` interaction styling. This is where classes would be really helpful. Consider creating a class to apply it to all of your button projects.

Screen Shot 2022-06-29 at 18 06 15

File: index.html [hunk 2]
```diff
@@ -32,5 +32,127 @@ <h1 class="playfair">Okolo <br> Oshoke</h1>
...
+            <h3>Skills</h3>
+            <ul>
+                <li>Codekit</li>
+                <li>Codepen</li>
+                <li>GitHub</li>
+                <li>GitLAb</li>
+                <li>Terminal</li>
+            </ul>
+            <img src="img/skills.svg" alt="">
+        </div>
```
Comment (vechicin) [inline index.html]: Please make sure the technology categories are wrapped in a `<ul>`, each of them in an `<li>`.

Screen Shot 2022-06-29 at 18 05 51

File: CSS/Styles.css [hunk 3]
```diff
@@ -109,4 +110,251 @@ body {
...
+
+  .skills {
+    background-color: #febf52;
+  }
+
+  .about-entry > h3 {
+    margin-top: 50px;
+    margin-left: 50px;
+    margin-bottom: 0;
+  }
```
Comment (vechicin) [inline CSS/Styles.css]: The text in this entire section doesn't have the same weight as in the template design. Please fix it.

Screen Shot 2022-06-29 at 18 08 23

File: CSS/Styles.css [hunk 4]
```diff
@@ -1,3 +1,4 @@
+/* stylelint-disable no-descending-specificity */
 *,
```
Comment (vechicin) [inline CSS/Styles.css:2]: Your design has a margin to both sides that is not present in the template design. Please fix them.

Screen Shot 2022-06-29 at 18 10 31

Screen Shot 2022-06-29 at 18 10 50

**Review by vechicin — CHANGES_REQUESTED** [review]
# Hi @lorenzoworx ✋, 

Good job so far! 🔥 
There are some issues that you still need to work on to go to the next project but you are almost there! 🚨 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights ⚡ 

- [x] Organized codebase
- [x] Good job overall following the template design

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 📂 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: CSS/Styles.css (hunk 4 above)
Comment (lorenzoworx) [reply CSS/Styles.css:2]: @vechicin Could you please clarify this for me? There's no margin on my design or on the figma template

**Review by KayLemba — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

Hello  @lorenzoworx       , 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

HIGHLIGHTS:

- Your project is professional 
- Your repo is professional 
- Well structured files 


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 

well-1

[OPTIONAL] Kindly add a screenshot to your README file showing your webpage user Interface. You can use extentions like fireshot to capturer the entire page.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**